src/middlewared/middlewared/plugins/audit/utils.py

```diff
@@ -24,6 +24,8 @@
     AuditEventParam.EVENT.value,
     AuditEventParam.SUCCESS.value,
 )
+AUDIT_CONTROLLER_SELECTIONS = ['MASTER', 'Active', 'BACKUP', 'Standby']
```

Comment:
Do you need all these options?

Reply:
Yes and no. 

The middleware routine `failover.status` reports the controllers as `MASTER` and `BACKUP`
_However..._
The standalone python script `hactl` translates those and reports the controller names as: `Active` and `Standby`.
The extra names are included to support both.